apps/personal/contacts/main.py

```diff
@@ -0,0 +1,334 @@
+# coding=utf-8
+import argparse
+import doctest
+import logging
+import pickle
+import os
+import vobject
+
+from ui.utils import flatten
+
+ZPUI_HOME = "~/.zpui/"  # todo : XDG_DATA_DIR/zpui
```

Comment:
The contacts app is going to be independent of ZPUI at some point, maybe use `.phone`?

Reply:
this is now the default